cmd/cli/util_test.go

```diff
@@ -93,7 +93,7 @@ var _ = Describe("Test getting pretty printed output of a list of meshes", func(
 		pp := getPrettyPrintedMeshInfoList(meshInfoList)
 
 		It("should have correct output", func() {
-			Expect(pp).To(Equal("\nMESH NAME\tNAMESPACE\tCONTROLLER PODS\tVERSION\tSMI SUPPORTED\n"))
+			Expect(pp).To(Equal("\nMESH NAME\tNAMESPACE\tCONTROLLER PODS\tVERSION\tSMI SUPPORTED\tADDED NAMESPACES\n"))
```

Comment:
Should we clarify this is the "MESH NAMESPACE" rather than just having NAMESPACE?